pkg/server/validateadmission.go

```diff
@@ -92,11 +92,30 @@ func (p *PipelineRunAdmission) HandleAdmission(review *admissionv1.AdmissionRevi
 				UID:     review.Request.UID,
 				Allowed: false,
 				Result: &metav1.Status{
-					Message: fmt.Sprintf("Builder (gotten %s) does not match AllowedBuilders (expected %s).", param.Name, buildstr),
+					Message: fmt.Sprintf("Builder (gotten %s) does not match AllowedBuilders (expected %s).", param.Value.StringVal, buildstr),
 				},
 			}
 			return nil
+		case "SOURCE_URL":
+			// permitted parameter, but not validated
+		case "USER_ID":
+			// permitted parameter, but not validated (TODO: should it?)
```

Comment:
I was thinking the same, but in order to not allow passing it we have to use a mutator right?

Reply:
Correct, I think that's out of scope for this patch